src/sass_types/value.h

```diff
@@ -1,41 +1,17 @@
 #ifndef SASS_TYPES_VALUE_H
 #define SASS_TYPES_VALUE_H
 
-#include <nan.h>
 #include <sass/values.h>
+#include "../common.h"
+#include <napi.h>
 
 namespace SassTypes
 {
   // This is the interface that all sass values must comply with
-  class Value : public Nan::ObjectWrap {
```

Comment:
We probable need to rebuild this without inheritance at all. Maybe this should become a template? We definitely need https://github.com/nodejs/node-addon-api/blob/master/doc/object_wrap.md here